pkg/agent/agent.go

```diff
@@ -496,14 +496,13 @@ func GetUserAgent() string {
 		agentName = agent.cfg.GetAgentName()
 		isGRPC = agent.cfg.IsUsingGRPC()
 	}
-	return util.FormatUserAgent(
+	return util.NewUserAgent(
```

Comment:
Remind me, once this goes out, that we will have to update the handling of the user agent for the amplify agents info

Reply:
Already have a story to update falconer as well to use the new useragent format